### ISSUE TITLE
Cannot build list:paired from selection of 4 pairs

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#11245]
![image](https://user-images.githubusercontent.com/458683/106166241-3eb07700-618c-11eb-9076-cea94dde75ec.png)

**Comment by jmchilton** [comment]
@hexylena do you happen to know if this a regression in 21.01?

**Comment by mvdbeek** [comment]
I don't think that has ever worked -- building collections of collections with the non-advanced collection builders.

**Comment by jmchilton** [comment]
I think it would be awesome if this did work, I just wanted to verify it isn't a regression. Thanks @mvdbeek
